client/webserver/site/src/js/markets.ts

```diff
@@ -1897,10 +1897,16 @@ export default class MarketsPage extends BasePage {
   handleOrderNote (note: OrderNote) {
     const order = note.order
     const mord = this.metaOrders[order.id]
-    // If metaOrder doesn't exist for the given order it means it was created
-    // via dexcctl and the GUI isn't aware of it or it was an inflight order.
-    // Call refreshActiveOrders to grab the order.
-    if (!mord || note.topic === 'AsyncOrderFailure') return this.refreshActiveOrders()
+    // - If metaOrder doesn't exist for the given order it means it was created
+    //  via dexcctl and the GUI isn't aware of it or it was an inflight order.
+    //  refreshActiveOrders must be called to grab this order.
+    // - If an OrderLoaded notification is recieved, it means an order that was
+    //   previously not "ready to tick" (due to its wallets not being connected
+    //   and unlocked) has now become ready to tick. The active orders section
```

Comment:
Doesn't this also happen when the user just logs in with active orders?  Won't it emit one of these OrderLoaded topic notes for each active order on initial login, causing it to refresh active orders multiple times?

Reply:
In those cases `readyToTick` will be false and the market page will not be loaded yet. I'll add a new notification topic though.

Reply:
Yes, but those will be before login is complete, so the market page will not yet be loaded.

Reply:
> Yes, but those will be before login is complete, so the market page will not yet be loaded.

Ah, OK, perfect.